module_evaluations/TCR-29_2023-09-21-mod-fqm.MD

```diff
@@ -0,0 +1,54 @@
+
+* [x] ACCEPTABLE
+* [x] ~INAPPLICABLE~
+* [ ] UNACCEPTABLE
+  * comments on what was evaluated/not evaluated, why a criterion failed
+
+## [Criteria](https://github.com/folio-org/tech-council/blob/7b10294a5c1c10c7e1a7c5b9f99f04bf07630f06/MODULE_ACCEPTANCE_CRITERIA.MD)
+
+## Shared/Common
+* [x] Uses Apache 2.0 license
+* [x] Module build MUST produce a valid module descriptor
+* [ ] Module descriptor MUST include interface requirements for all consumed APIs
+  - This module has implicit dependencies on other modules that are not expressed in the `requires` section of the module descriptor
+* [x] Third party dependencies use an Apache 2.0 compatible license
+* [x] Installation documentation is included
+* [x] Personal data form is completed, accurate, and provided as `PERSONAL_DATA_DISCLOSURE.md` file
+* [x] Sensitive and environment-specific information is not checked into git repository
+* [x] Module is written in a language and framework from the [officially approved technologies](https://wiki.folio.org/display/TC/Officially+Supported+Technologies) page
+* [x] Module only uses FOLIO interfaces already provided by previously accepted modules _e.g. a UI module cannot be accepted that relies on an interface only provided by a back end module that hasn't been accepted yet_
+* [x] Module gracefully handles the absence of third party systems or related configuration
+* [x] Sonarqube hasn't identified any security issues, major code smells or excessive (>3%) duplication
+* [x] Uses [officially supported](https://wiki.folio.org/display/TC/Officially+Supported+Technologies) build tools
+* [x] Unit tests have 80% coverage or greater, and are based on [officially approved technologies](https://wiki.folio.org/display/TC/Officially+Supported+Technologies)
+
+## Backend
+* [x] Module's repository includes a compliant Module Descriptor
+* [x] Module includes executable implementations of all endpoints in the provides section of the Module Descriptor
+* [x] Environment vars are documented in the ModuleDescriptor
+* [x] If a module provides interfaces intended to be consumed by other FOLIO Modules, they must be defined in the Module Descriptor "provides" section
+* [ ] All API endpoints are documented in RAML or OpenAPI
+    - The swagger documentation does not include: `GET: /entity-types/{entity-type-id`, `GET: /entity-types/{entity-type-id}/columns/{column-name}/values`, `GET: /query`, `GET: /query/{query-id}`, `GET: /query/{query-id}/sortedIds`, `POST: /query/contents`, `POST: /query`, and `DELETE: /query/{query-id}` 
```

Comment:
It's a little confusing, since mod-fqm-manager and edge-fqm share the same OpenAPI spec, but the OAS lives in a separate lib: https://github.com/folio-org/folio-query-tool-metadata/tree/master/src/main/resources/swagger.api